addon/appModules/notepad++/editWindow.py

```diff
@@ -126,7 +128,19 @@ def script_goToFirstOverflowingCharacter(self, gesture):
 	script_goToFirstOverflowingCharacter.category = "Notepad++"
 
 	def script_reportLineInfo(self, gesture):
-		ui.message(self.parent.next.next.firstChild.getChild(2).name) 
+		lineInfo = self.parent.next.next.firstChild.getChild(2).name
+		# Get only the numbers we want from the statusBar.
+		lineInfoExpression = re.compile(r"^Ln\D+(\d\w*)\D+(\d\w*)\D+(\d\w*)\D+(\d\w*)")
+		lines, columns, selectedCharacters, selectedLines = lineInfoExpression.match(lineInfo).groups()
+		#Translators: The line and column position of the cursor.
```

Comment:
Localisation could certainly be a problem here, but I'm guessing the regexp won't match in that case. You should definitely handle the possibility that this doesn't match.

If you want to support Notepad++ in other languages, you'll need to provide a way to check for multiple regexps here. You could always do that later though depending on demand. Do not use gettext for this, though.